NetCord/ApplicationCommandInteractionData.cs

```diff
@@ -1,10 +1,22 @@
 ﻿namespace NetCord;
 
+/// <summary>
+/// Contains information on an <see cref="ApplicationCommandInteraction"/>.
+/// </summary>
 public class ApplicationCommandInteractionData(JsonModels.JsonInteractionData jsonModel) : InteractionData(jsonModel)
 {
+    /// <summary>
+    /// The parent <see cref="ApplicationCommandInteraction"/>'s ID.
+    /// </summary>
     public ulong Id => _jsonModel.Id.GetValueOrDefault();
 
+    /// <summary>
+    /// The parent <see cref="ApplicationCommandInteraction"/>'s name.
+    /// </summary>
     public string Name => _jsonModel.Name!;
 
+    /// <summary>
+    /// The parent <see cref="ApplicationCommandInteraction"/>'s type.
+    /// </summary>
```

Comment:
The information of interaction are stored in the interaction I believe. Here the values are for the application command itself I think.